subprojects/workers/src/main/java/org/gradle/workers/internal/DefaultWorkerConfiguration.java

```diff
@@ -59,6 +61,16 @@ public void setParams(Serializable[] params) {
     }
 
     @Override
+    public ForkMode getForkMode() {
+        return forkMode;
+    }
+
+    @Override
+    public void setForkMode(ForkMode fork) {
+        this.forkMode = fork == null ? ForkMode.AUTO : fork;
```

Comment:
So setting it to `null` reset it to the default. Otherwise we would have to annotate `forkMode` with `@Nullable` and handle `null` when consuming it.